classes/class-wpcom-liveblog-wp-cli.php

```diff
@@ -85,16 +114,16 @@ public function fix_archive() {
 			// find comment_ids that NEED to be replaced
 			$entries_replace = $wpdb->get_results(
 				$wpdb->prepare(
-					"SELECT DISTINCT meta_value 
-					FROM $wpdb->commentmeta 
-					INNER JOIN $wpdb->comments 
+					"SELECT DISTINCT meta_value
+					FROM $wpdb->commentmeta
+					INNER JOIN $wpdb->comments
 					ON $wpdb->comments.comment_id = $wpdb->commentmeta.comment_id
 					WHERE comment_post_id = %d
 					ORDER BY meta_value ASC", $post_id )
 				);
 
-			// check to make sure entry content being replaced matches available
-			if ( count( $entries_replace ) === count( $correct_contents) ) {
+			// check to make sure entry content being replaced matches available and that this is not a dry run
+			if ( count( $entries_replace ) === count( $correct_contents) && false === $is_dryrun ) {
```

Comment:
Let's move this dry run check until just before the update action, so we can better report on what will be done in live mode.